stable/nats/templates/deployment.yaml

```diff
@@ -0,0 +1,162 @@
+{{ toYaml .Values.nodeSelector | indent 8 }}
+      {{- end }}
+      {{- if .Values.tolerations }}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+      {{- end }}
+      {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName | quote }}
+      {{- end }}
+      {{- if eq .Values.antiAffinity "hard" }}
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - topologyKey: "kubernetes.io/hostname"
+              labelSelector:
+                matchLabels:
+                  app: "{{ template "nats.name" . }}"
+                  release: {{ .Release.Name | quote }}
+      {{- else if eq .Values.antiAffinity "soft" }}
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 1
+            podAffinityTerm:
+              topologyKey: kubernetes.io/hostname
+              labelSelector:
+                matchLabels:
+                  app: "{{ template "nats.name" . }}"
+                  release: {{ .Release.Name | quote }}
+      {{- end }}
+      containers:
+      - name: {{ template "nats.name" . }}
+        image: {{ template "nats.image" . }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command:
+        - gnatsd
+        args:
+        - -c
+        - /opt/nats/gnatsd.conf
+        {{- if .Values.extraArgs }}
+{{ toYaml .Values.extraArgs | indent 8 }}
+        {{- end }}
+        ports:
+        - name: client
+          containerPort: {{ .Values.client.service.port }}
+        - name: cluster
+          containerPort: {{ .Values.cluster.service.port }}
+        - name: monitoring
+          containerPort: {{ .Values.monitoring.service.port }}
+        {{- if .Values.livenessProbe.enabled }}
+        livenessProbe:
+          httpGet:
+            path: /
+            port: monitoring
+          initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.livenessProbe.successThreshold }}
+          failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+        {{- end }}
+        {{- if .Values.readinessProbe.enabled }}
+        readinessProbe:
+          httpGet:
+            path: /
+            port: monitoring
+          initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+          periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
+          timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
+          successThreshold: {{ .Values.readinessProbe.successThreshold }}
+          failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
+        {{- end }}
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
+        volumeMounts:
+          - name: config
+            mountPath: /opt/nats/gnatsd.conf
```

Comment:
Why? It's just a path for configuration, also it have been removed it in #13505 already.

Reason for the change, not all user will apply image: `docker.io/bitnami/nats`
It is weird to use`docker.io/nats:1.4.1` but mount on `/opt/bitnami/nats/gnatsd.conf`

Reply:
@tompizmor may need your opinion on this change.

Reply:
Because the directory **/opt/bitnami/nats/** is the "working directory" and the non-root user (1001 by default) has writing permission on that directory. See: https://github.com/bitnami/bitnami-docker-nats/blob/master/1/debian-9/Dockerfile#L12

Reply:
got it, will revert #13505 later